include/usb/usb_cdc.h

```diff
@@ -0,0 +1,164 @@
+/* usb_cdc.h - USB CDC-ACM and CDC-ECM public header */
+
+/*
+ * Copyright (c) 2017 PHYTEC Messtechnik GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+/**
+ * @file
+ * @brief USB Communicatons Device Class (CDC) public header
+ *
+ * Header follows the Class Definitions for
+ * Communications Devices Specification (CDC120-20101103-track.pdf),
+ * PSTN Devices Specification (PSTN120.pdf) and
+ * Ethernet Control Model Devices Specification (ECM120.pdf).
+ * Header is limited to ACM and ECM Subclasses.
+ */
+
+#ifndef __USB_CDC_H__
+#define __USB_CDC_H__
+
+/** CDC Specification release number in BCD format */
+#define CDC_SRN_1_20			0x0120
+
+/** CDC Class Code */
+#define COMMUNICATION_DEVICE_CLASS	0x02
+
+/** Communications Class Subclass Codes */
+#define ACM_SUBCLASS			0x02
+#define ECM_SUBCLASS			0x06
+
+/** Communications Class Protocol Codes */
+#define AT_CMD_V250_PROTOCOL		0x01
+
+/**
+ * @brief Data Class Interface Codes
+ * @note CDC120-20101103-track.pdf, 4.5, Table 6
+ */
+#define DATA_INTERFACE_CLASS		0x0A
+
+/**
+ * @brief Values for the bDescriptorType Field
+ * @note CDC120-20101103-track.pdf, 5.2.3, Table 12
+ */
+#define CS_INTERFACE			0x24
+#define CS_ENDPOINT			0x25
+
+/**
+ * @brief bDescriptor SubType for Communications
+ * Class Functional Descriptors
+ * @note CDC120-20101103-track.pdf, 5.2.3, Table 13
+ */
+#define HEADER_FUNC_DESC		0x00
+#define CALL_MANAGEMENT_FUNC_DESC	0x01
+#define ACM_FUNC_DESC			0x02
+#define UNION_FUNC_DESC			0x06
+#define ETHERNET_FUNC_DESC		0x0F
+
```

Comment:
Just realized we have already those definitions in subsys/usb/class/cdc_acm.h. So if you want to rename them would it make sense to remove old ones replacing them in the code?

Reply:
For this PR, subsys/usb/class/cdc_acm.h is there for webusb sample only. Webusb needs much more work, I have only made the header for WEBUSB standard.

Reply:
Whoever source subsys/usb/class/cdc_acm.h should source new header then, right?

Reply:
No for what? It can be removed later when webusb is moved to subsys/usb/class

Reply:
OK, can be done this way, do not forget then to remove those redundant definitions.